staging/src/k8s.io/apimachinery/pkg/util/strategicpatch/patch_test.go

```diff
@@ -1817,7 +1821,31 @@ testCases:
               other: b
         - name: 2
           other: b
-`)
+  - description: defined null values should propagate overwrite current fields (with conflict)
+    original:
+      name: 2
+    twoWay:
+      name: 1
+      value: 1
+      other: null
+    modified:
+      name: 1
+      value: 1
+      other: null
+    current:
+      name: a
+      other: a
+    threeWay:
+      name: 1
+      value: 1
+      other: null
+    result:
+      name: 1
+      value: 1
+    twoWayResult:
```

Comment:
tiny comment: do you thing `twoWayResult` could be moved under `twoWay` just for better readability?